templates/zerver/help/move-a-topic.md

```diff
@@ -0,0 +1,17 @@
+# Move a topic
+
+Just admin users can move topics between two public streams.
+
+### Move a topic
+
+{start_tabs}
+
+1. Hover over a topic in the left sidebar.
+
+1. Click the down chevron (<i class="fa fa-chevron-down"></i>) to the right.
+
+1. Select the last option. It may be called **Move all messages in <topic name\> to...**
+
+1. Select the destiny stream where the topic will be moved.
+
+{end_tabs}
```

Comment:
I think probably we should make this a section of the existing "Rename a topic" article, rather than it's own article.  We might then decide to rename the new article as well to e.g. "Moving topics".

Reply:
Sure, let me see how to do that :-)